curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/quoting-strings-with-single-quotes.md

```diff
@@ -26,9 +26,11 @@ However, this becomes a problem if you need to use the outermost quotes within i
 
 ```js
 goodStr = 'Jake asks Finn, "Hey, let\'s go on an adventure?"'; 
-badStr = 'Finn responds, "Let's go!"'; // Throws an error
+badStr = 'Finn responds, "Let's go!"';
 ```
 
+Here `badStr` will throw an error.
+
 In the <dfn>goodStr</dfn> above, you can use both quotes safely by using the backslash <code>\\</code> as an escape character.
 
 **Note:** The backslash <code>\\</code> should not be confused with the forward slash `/`. They do not do the same thing.
```

Comment:
Can this be changed, as it relates to the _work-around_ from Crowdin which I believe is now fixed?

Reply:
Ah yes, thank you.